src/definitions/collections/menu.less

```diff
@@ -297,6 +307,7 @@
     }
     .ui.vertical.menu .dropdown.item.upward .menu {
         bottom: 0;
+        top: auto !important;
```

Comment:
Could !important get replaced by increasing .ui specificity?

Reply:
There are also ```!important``` flags that are not part of my PR which wouldn't allow to override with ```.ui``` specificity.

I would have to refactor and remove them too.

I feel like we've too many bloated CSS blocks in several modules that need to be removed.

Since we dropped the road map for FUI v3, we should consider to refactor this version to keep as much as clean and reduce the file size.

I would find the time for refactoring the CSS first. Then we can consider the refactoring JS modules.

Reply:
I couldn't find any solution unless we wrap the text inside dropdown item with extra container like `<span class="text">Item</span>`.

Please check this fiddle: https://jsfiddle.net/ko2in/twpkudxs/1/

As you can see, it just work without any CSS modifications.

Reply:
> Should I discard this PR or keep it for another major release? What do you think @lubber-de ?

Let's keep it (as your work is still very good!), as planned for 2.10, and take a look again later. I label that on hold until then and convert to draft.